include/wx/frame.h

```diff
@@ -233,6 +233,9 @@ class WXDLLIMPEXP_CORE wxFrameBase : public wxTopLevelWindow
     // return false if there is no help for such item
     bool ShowMenuHelp(int helpid);
 
+    // show the help string for a menu item directly without finding it by id
+    bool ShowMenuHelp(const wxString& helpString);
```

Comment:
I'm not really sure we need this function, it seems to be exactly the same as `DoGiveHelp()`, isn't it?

Reply:
You're right. Of course DoGiveHelp can be used directly here.